phpBB/styles/prosilver/template/forumlist_body.html

```diff
@@ -11,98 +11,125 @@
 	<!-- IF forumrow.S_IS_CAT or forumrow.S_FIRST_ROW  or forumrow.S_NO_CAT  -->
 		<div class="forabg">
 			<div class="inner">
-			<ul class="topiclist">
-				<li class="header">
-					<!-- EVENT forumlist_body_category_header_row_prepend -->
-					<dl class="icon">
-						<dt><div class="list-inner"><!-- IF forumrow.S_IS_CAT --><a href="{forumrow.U_VIEWFORUM}">{forumrow.FORUM_NAME}</a><!-- ELSE -->{L_FORUM}<!-- ENDIF --></div></dt>
-						<dd class="topics">{L_TOPICS}</dd>
-						<dd class="posts">{L_POSTS}</dd>
-						<dd class="lastpost"><span>{L_LAST_POST}</span></dd>
-					</dl>
-					<!-- EVENT forumlist_body_category_header_row_append -->
-				</li>
-			</ul>
-			<ul class="topiclist forums">
+				<ul class="topiclist">
```

Comment:
I know the ?w=* thing,
but that is not what style authors will get when trying to update their style.